docs/man/nng_http_req_del_header.3http.adoc

```diff
@@ -11,7 +11,7 @@
 
 == NAME
 
-nng_http_req_set_header - set HTTP request header
+nng_http_req_del_header - del HTTP request header
```

Comment:
Let's change this to proper English -- "delete HTTP request header" (spell out delete fully).